src/main/java/com/fasterxml/jackson/databind/jsontype/impl/AsPropertyTypeDeserializer.java

```diff
@@ -39,23 +46,25 @@ public class AsPropertyTypeDeserializer extends AsArrayTypeDeserializer
     public AsPropertyTypeDeserializer(JavaType bt, TypeIdResolver idRes,
             String typePropertyName, boolean typeIdVisible, JavaType defaultImpl)
     {
-        this(bt, idRes, typePropertyName, typeIdVisible, defaultImpl, As.PROPERTY);
+        this(bt, idRes, typePropertyName, typeIdVisible, defaultImpl, As.PROPERTY, Boolean.FALSE);
     }
 
     /**
      * @since 2.8
      */
     public AsPropertyTypeDeserializer(JavaType bt, TypeIdResolver idRes,
             String typePropertyName, boolean typeIdVisible, JavaType defaultImpl,
-            As inclusion)
+            As inclusion, Boolean hasTypeResolverAttached)
     {
         super(bt, idRes, typePropertyName, typeIdVisible, defaultImpl);
         _inclusion = inclusion;
```

Comment:
this is wrong - you modifed its param list - you need to keep the old `@since 2.8` and add a new `@since 2.15` one with the new hasTypeResolverAttached param 

does it have to be `Boolean`? This type is very unpopular - it confuses developers who don't recognise that it is not the same as `boolean` -- allows `null`